pkg/skaffold/build/jib/gradle.go

```diff
@@ -97,8 +98,8 @@ func getSyncMapCommandGradle(ctx context.Context, workspace string, a *latest.Ji
 }
 
 // GenerateGradleBuildArgs generates the arguments to Gradle for building the project as an image.
-func GenerateGradleBuildArgs(task string, imageName string, a *latest.JibArtifact, skipTests, pushImages bool, deps []*latest.ArtifactDependency, r ArtifactResolver, insecureRegistries map[string]bool) []string {
-	args := gradleBuildArgsFunc(task, a, skipTests, true, MinimumJibGradleVersion)
+func GenerateGradleBuildArgs(task string, imageName string, a *latest.JibArtifact, skipTests, pushImages bool, deps []*latest.ArtifactDependency, r ArtifactResolver, insecureRegistries map[string]bool, showColors bool) []string {
+	args := gradleBuildArgsFunc(task, a, skipTests, showColors, MinimumJibGradleVersion)
```

Comment:
super nit: `colorOutput` instead of `showColors` ?

Reply:
https://github.com/GoogleContainerTools/skaffold/blob/8caebb9955417b706a092bb7f896e0b544249797/pkg/skaffold/build/jib/gradle.go#L114
Many of the tests and this line use ```showColors```, so i'll leave it be to stay consistent. I think it is clearer with ```showColors``` as well since in ```colorOutput```  "color" can either be an adjective or a noun.